src/components/chat/comment-textbox.tsx

```diff
@@ -108,6 +130,30 @@ export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedCommen
         onChange={handleCommentTextAreaChange}
         onKeyDown={handleCommentTextboxKeyDown}
       />
+      {
+        showCommentTag && commentTags &&
+        <select
+          ref={selectElt}
+          data-test="comment-textbox-dropdown"
+          onChange={(e) => {
+            handleSelectDropDown(e.target.value);
+          }}
+        >
+          {
+            defaultTag &&
+            <option key={"sel-ss"} value={defaultTag}> { defaultTag } </option>
+          }
+          {
+            Object.keys(commentTags).map(key => {
+              const value = commentTags[key as keyof typeof commentTags];
```

Comment:
```suggestion
              const value = commentTags[key];
```
when you remove the `| never[]` from the definition, you shouldn't need the cast.